kinetic/distribution.yaml

```diff
@@ -15759,6 +15759,20 @@ repositories:
       url: https://github.com/NicksSimulationsROS/sync_params.git
       version: ros-kinetic
     status: developed
+  t_mapping:
+    doc:
+      type: git
+      url: https://github.com/mitre/t_mapping.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/tbostic32/t_mapping-release.git
```

Comment:
It's a little weird to have the release repository under a personal account while the source is under an organization.  I'd suggest moving this repository to the `mitre` organization on GitHub, though it is not required.